plugins/react-kit/index.js

```diff
@@ -10,5 +10,6 @@ module.exports = {
         require('@reframe/start'),
         require('@reframe/eject'),
         require('@reframe/build'),
+        require('@reframe/aws'),
```

Comment:
The react-kit is agnostic of the deployment strategy. E.g. the frontend-only starter uses `@reframe/deploy-git` for static host deployment. We could add `@reframe/aws` to the `react-app` starter. Feel free to object